src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -3,12 +3,22 @@ class Webui::MonitorController < Webui::WebuiController
   before_action :require_available_architectures, only: [:index]
   before_action :fetch_workerstatus, only: [:old, :filtered_list, :update_building]
 
-  def fetch_workerstatus
-    @workerstatus = WorkerStatus.hidden.to_hash
+  def self.addarrays(arr1, arr2)
```

Comment:
Similar here. One would need a private_class_method or private inside a class << self block to make this effective.

I am currently checking if doing so would break anything. Stay tuned.